packages/socket-mode/test/integration.spec.js

```diff
@@ -0,0 +1,101 @@
+const { assert } = require('chai');
+const { SocketModeClient, LogLevel } = require('../.');
+const { WebSocketServer} = require('ws');
+const { createServer } = require('http');
+
+const HTTP_PORT = 8080;
```

Comment:
We may want to avoid 8080 port for testing on our local machines because Zscaler and other similar products could use this port...